src/pages/documentation/faq.md

```diff
@@ -111,6 +111,52 @@ Add the following rule to your custom ProGuard rules file, typically labeled `pr
 }
 ```
 
+### Is there a change in minimum API level supported by Android SDK?
```

Comment:
Can we rename all the `Android SDK` occurrences to one of these options: 
`Android Platform Mobile SDK`, `Android Mobile SDK`, `Mobile SDK for Android`